recipes/lytekit/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "lytekit" %}
+{% set version = "0.15.27" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/lytekit-{{ version }}.tar.gz
+  sha256: bf3d9e127ad02b94e88c9b7f4f8df2e81946104c470c935d00ad99b312f46915
+
+build:
+  skip: true  # [py<37 or win or py>310]
```

Comment:
Looks like this one can be noarch.

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.